src/core/managers/internal/NotificationsFeedManager/types/transaction.types.ts

```diff
@@ -0,0 +1,34 @@
+import type { TransactionsSliceType } from 'store/slices/transactions/transactionsSlice.types';
+import type { ServerTransactionType } from 'types/serverTransactions.types';
+
+export interface TransactionListItem {
+  title: string;
+  amount: string;
+  mainIconUrl?: string;
+  details?: {
+    directionLabel: string;
+    initiator: string;
+    iconUrl?: string;
+  };
+  rightIcons?: string[];
+}
+
+export interface GetHistoricalTransactionsParamsType {
```

Comment:
these two interfaces could extend a base one
are they not similar to something in transactions list ?
maybe sit in utils/transactions or in general types ?